ext/RastersCoordinateTransformationsExt/affineprojected.jl

```diff
@@ -4,7 +4,7 @@ function AffineProjected(f;
     AffineProjected(f, data, metadata, crs, mappedcrs, paired_lookup, dim)
 end
 
-DD.dim(lookup::AffineProjected) = lookup.dim
+DD.dims(lookup::AffineProjected) = lookup.dim
```

Comment:
```suggestion
DD.dim(lookup::AffineProjected) = lookup.dim
```

This one isn't a typo! 

Maybe its silly to have but its only for lookups and at the `Lookup` module level `dims` is not available - and it really shouldn't be a Lookup module function. `dims` also returns a Tuple that we don't have for a lookup.

Reply:
Yeah I got confused because when I load DimensionalData I didn't find DD.dim